src/routes/safe/components/Apps/components/ConfirmTransactionModal.tsx

```diff
@@ -24,6 +24,7 @@ import { estimateSafeTxGas } from 'src/logic/safe/transactions/gas'
 
 import GasEstimationInfo from './GasEstimationInfo'
 import { getNetworkInfo } from 'src/config'
+import { TransactionParams } from '..'
```

Comment:
I think I prefer the same approach as in `useIframeMessageHandler.ts` setting the route with ../index